server/tests/test_mpesa_routes.py

```
@@ -0,0 +1,314 @@
+from models.payment import MpesaTransaction
+from models.user import User
+
+
+@pytest.fixture
+def auth_headers(client, session):
+    """Fixture to get JWT auth headers for authenticated requests"""
+    # Create a test user first
+    test_user = User(
+        username="testuser",
+        email="test@example.com",
+        password="testpassword"
```

Ruff (0.13.1)

Possible hardcoded password assigned to argument: "password"

(S106)